File: csharp/1523.count-odd-numbers-in-an-interval-range.cs
/*
 * @lc app=leetcode id=1523 lang=csharp
 *
 * [1523] Count Odd Numbers in an Interval Range
 */

// @lc code=start
public class Solution {
    public int CountOdds(int low, int high) {
        return (high + 1) / 2 - low / 2;
    }
}
// @lc code=end

File: java/1523.count-odd-numbers-in-an-interval-range.java
/*
 * @lc app=leetcode id=1523 lang=java
 *
 * [1523] Count Odd Numbers in an Interval Range
 */

// @lc code=start
class Solution {
    public int countOdds(int low, int high) {
        return ((low % 2) | (high % 2)) + (high - low) / 2;
    }
}
// @lc code=end

